.favorites.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "type": "Directory",
+        "name": "/Users/asher/projects/charts/stable/prometheus-operator",
+        "parent_id": null,
+        "workspaceRoot": "/Users/asher/projects/charts",
+        "workspacePath": "stable/prometheus-operator",
+        "id": "Z6ebXlor8sOCqjPt"
+    }
+]
```

Comment:
This file should not be in the pr.
Otherwise looks good.

Reply:
🤦‍♂ fixed....